QUEST_LV_0300.tsv

```diff
@@ -143,10 +143,10 @@ QUEST_LV_0300_20150729_000142	Now you've collected them all. Here, I'll give you
 QUEST_LV_0300_20150729_000143	I just want to rest...
 QUEST_LV_0300_20150729_000144	Is this the last time? Really.
 QUEST_LV_0300_20150729_000145	Let's see if we have all the detonators.
-QUEST_LV_0300_20150729_000146	You've came here... Goodbye for now.
+QUEST_LV_0300_20150729_000146	You've came... but it's already time for me to go.
 QUEST_LV_0300_20150729_000147	I feel drained.
 QUEST_LV_0300_20150729_000148	Yes. That's it. Give it to me.
-QUEST_LV_0300_20150729_000149	Notice/!/When touched with the device, I feel like my soul drained!{nl}I have to find someone who is able to relase this situation!
+QUEST_LV_0300_20150729_000149	Notice/!/When I touch the device, it feels like my soul is being drained!{nl}I have to find someone who is able to fix this situation!
```

Comment:
For these Notice/!/, we settled for the convention of using "you" instead of I. So it's:

Notice/!/When you touch the device, it feels like your soul is being drained!{nl}You must find someone who can solve this predicament/difficult situation/issue! <--your choice on which word to use